src/panels/lovelace/components/notifications/hui-persistent-notification-item.js

```diff
@@ -15,11 +16,24 @@ import LocalizeMixin from '../../../../mixins/localize-mixin.js';
 export class HuiPersistentNotificationItem extends LocalizeMixin(PolymerElement) {
   static get template() {
     return html`
+    <style>
+      ha-relative-time {
+        color: var(--secondary-text-color);
+        display: block;
+        margin-top: 6px;
+        text-align: right;
+      }
+    </style>
     <hui-notification-item-template>
       <span slot="header">[[_computeTitle(notification)]]</span>
       
       <ha-markdown content="[[notification.message]]"></ha-markdown>
       
+      <ha-relative-time
```

Comment:
Please wrap this in a paper-tooltip that shows the actual formatted date when hovered. [Example](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/panels/config/config-entries/ha-config-entries-dashboard.js#L91)